Sming/Makefile-project.mk

```diff
@@ -322,8 +325,14 @@ $(TARGET_OUT): $(APP_AR)
 	
 	$(vecho) "------------------------------------------------------------------------------"
 ifeq ($(UNAME),Windows)
-	$(vecho) "Memory layout info:"
-	$(Q) $(SDK_TOOLS)/memanalyzer.exe $(OBJDUMP).exe $@
+	$(vecho) "Memory layout info:";
+#Redirect to file for keeping track over multiple git revisions	
+#Check for existing meminfo file
+	$(Q) if [ -f "$(FW_MEMINFO_OLD)" ]; then \
+    	mv $(FW_MEMINFO_OLD) $(FW_MEMINFO).old; \
```

Comment:
No, fwMeminfo.txt and .old are in the out/firmware wich gets deleted by make clean. On the other hand clean target will also save fwMeminfo into FW_MEMINFO_OLD which is in /out 

The cycle is this
1. make all -> /out/firmware/fwMeminfo.txt
2. make clean -> move /out/firmware/fwMeminfo.txt into /out + remove /out/firmware
3. make all -> (if_exist) move /out/fwMeminfo.old into /out/firmware/fwinfo.txt.old; overwrite /out/firmware/fwMeminfo.txt
4. ...

Even if you don't do make clean you will always have 2 files because there are file moves not file copy. 

Then I use beyondCompare to see how latest code changes affected the binary. I also added /out/firmware to my project's local gitignore so i can track meminfo over releases to see where it went wrong. 

What would be really nice: use perl or something to take the hex difference between the old and new file and output to console as part of make process but it's overcomplicated i guess. 
Something like : "memanalyzer.exe > fwMeminfo.new" then "perl perlscript fwMeminfo.new fwMeminfo.old" -> this will output memanalyzer data plus variations in brackets something like
   Section|                   Description| Start (hex)|   End (hex)|Used space
------------------------------------------------------------------------------
      data|        Initialized Data (RAM)|    3FFE8000|    3FFE85C8|    1480 (+20)
    rodata|           ReadOnly Data (RAM)|    3FFE85D0|    3FFEA218|    7240 (+8) 
       bss|      Uninitialized Data (RAM)|    3FFEA218|    3FFF3468|   37456 (+545)
      text|            Cached Code (IRAM)|    40100000|    401054C1|   21697  (+0)
irom0_text|           Uncached Code (SPI)|    40209000|    4023B8E6|  207078 (-33)
Total Used RAM : 46176 (+652)
Free RAM : 35744	(-500)
Free IRam : 11089   (-152)